src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -260,5 +261,16 @@ public async Task<IActionResult> ValidateUlnOverlap([FromBody] ValidateUlnOverla
 
             return Ok(response);
         }
+
+        [HttpGet]
+        [Route("validate")]
+        public async Task<IActionResult> ValidateApprenticeship([FromQuery] string firstName, [FromQuery] string lastName, [FromQuery] DateTime dateOfBirth)
+        {
+            var query = new GetApprenticeshipsValidateQuery(firstName, lastName, dateOfBirth);
+
+            var result = await _mediator.Send(query);
+
+            return Ok(result);
```

Comment:
The query result should be mapped to a separate response type, rather than being exposed through the controller result directly.

Reply:
The result is not exposing any entities, it is a model so should not be a prob. Resolving this as discussed.